apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceHeaderContext.cs

```diff
@@ -28,6 +28,8 @@ namespace Google.Cloud.Diagnostics.Common
     /// </remarks>
     internal sealed class TraceHeaderContext
     {
+        private static readonly TraceIdFactory _traceIdFactory = TraceIdFactory.Create();
```

Comment:
Oh I see.  It's used in a few places so I think it's best to keep it separate.  Although the class itself is very simple, it just creates a new GUID.